src/frontend/js/widgets/Dashboard/components/DashboardItem/Certificate/index.tsx

```diff
@@ -20,40 +21,43 @@ export const DashboardItemCertificate = ({
   productType,
   mode,
 }: DashboardItemCertificateProps) => {
-  if (certificate) {
-    if (certificateDefinition) {
-      throw new Error('certificate and certificateDefinition are mutually exclusive');
+  const getCourseFromCertificate = () => {
```

Comment:
In the future we will maybe have to make this function available to other components, WDYT about make a standalone function? And then writing a unit test forit?

Reply:
Ok, I created a `CertificateHelper` implementing a `getCourse` method. Let's take a look to the dedicated commit.